main.js

```diff
@@ -18,8 +18,11 @@
  *        MIT License. All code unless otherwise specified is
  *        Copyright (c) Matthew Knox and Contributors 2015.
  */
-
+'use strict';
 // Load NodeJS Modifications/Variables
+require('babel-core');
+require('babel-register');
+require('babel-polyfill');
```

Comment:
The issue the require hook and polyfill must be in the top level file, in this case main.js, so that all future requires have the es6 transcoder